src/wasm/compat/types/mount.rs

```diff
@@ -621,15 +625,19 @@ impl WasmMount {
             let cids = file
                 .get_cids(&fs.forest, &self.metadata_blockstore)
                 .await
-                .ok()
-                .ok_or(TombWasmError::new("retrieve CIDs"))?;
-            api_blockstore.find_cids(cids).await.ok();
+                .map_err(|_| TombWasmError::new("retrieve CIDs"))?;
+            api_blockstore
+                .find_cids(cids)
+                .await
+                .map_err(to_wasm_error_with_msg("find_cids"))?;
         }
 
         info!("read_bytes() running fs.read @ {:?}", path_segments);
 
+        // Attempt to fetch from local first, remote second
+        let split_store = DoubleSplitStore::new(&self.content_blockstore, &api_blockstore);
```

Comment:
this fixes the `write_read` test in web.rs and i think is a good idea to fallback on in our memory `BlockStore` first before seeking the network's help. that said the fact that the `write_read` test fails without this implementation makes me worry that something is still wrong with the api blockstore.